app/settings/tokens/edit/template.hbs

```diff
@@ -32,19 +32,21 @@
             {{t 'settings.tokens.createSuccess.instructions'}}
         </p>
         <Button
-            {{on 'click' (action this.refresh)}}
+            {{on 'click' (perform this.refreshToken)}}
```

Comment:
Similarly, this can be reverted back to use `(action)` as well

Reply:
Ok, this had resolved the local build issues we were experiencing. If it works otherwise, I have restored it to the action.